### PR TITLE
Display weather info

### PR DESCRIPTION
**Pull request by venkykatta**
**Weather-information**

### CODE REVIEW

**Comment by su66uu** [comment]
Please post some result screenshots

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/home/shikhar/flask-intro/env/bin/python3.7"
+}
```
Comment (su66uu) [inline .vscode/settings.json]: It is not good practice to maintain editor settings on VCS. Please clean it up

Comment (su66uu) [reply .vscode/settings.json]: Remove the _pycache_ also

File: template/static/index.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+function handle_form()
+{
+  alert("Submit button pressed");
```
Comment (su66uu) [inline template/static/index.js]: You are not using file anywhere. Do you need it?

**Review by su66uu — CHANGES_REQUESTED** [review]

**Comment by su66uu** [comment]
Seems this code won't work. Please check what's wrong in it?

File: .vscode/settings.json (hunk 1 above)
Comment (venkykatta) [reply .vscode/settings.json]: okay,  i deleted this .vscode

File: template/static/index.js (hunk 2 above)
Comment (venkykatta) [reply template/static/index.js]: yes, the file index.js was not used anywhere, so there is no need of it

Comment (venkykatta) [reply template/static/index.js]: And also remove the pycahe

Comment (su66uu) [reply template/static/index.js]: Can you delete this then?

Comment (venkykatta) [reply template/static/index.js]: yes, i delete this one

File: app.py [hunk 3]
```diff
@@ -1,8 +1,27 @@
...
-    return "Hello, World!"
+@app.route('/')
+def index():
+    return render_template("index.html")    
+
+
+@app.route('/', methods=['POST'])
+def Onsubmit():
+    place = request.form['place']
+    # place_in_uppercase = place.upper()
```
Comment (su66uu) [inline app.py]: Don't keep the commented code with the working code

File: app.py [hunk 4]
```diff
@@ -1,8 +1,27 @@
...
+    API_KEY = 'e03b0e4011564a1a9e11eb7086d70d1e'
+    url = f'http://api.openweathermap.org/data/2.5/weather?q={place}&APPID={API_KEY}&units=metric'
+    weather_info = requests.get(url).json()
+    temparature = weather_info["main"]["temp"]
+    temp_in_string = str(temparature)
+    # return f'The current temparature in {place_in_uppercase} is {temp_in_string} \u00b0C'
```
Comment (su66uu) [inline app.py]: Here also remove it

**Review by su66uu — CHANGES_REQUESTED** [review]

File: app.py (hunk 4 above)
Comment (venkykatta) [reply app.py]: yeah sure.

File: app.py (hunk 3 above)
Comment (venkykatta) [reply app.py]: okay, i don't do again this type of fault.Thank you

File: template/index.html [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+                <input class="input is rounded" type="text" id="place" name="place" placeholder="Enter place name">
+            </div>
+            <div class="weather-input-submit"> </div>
+            <div class="field is-grouped">
+      <div class="control">
+        <button class="button is-link">Submit</button>
+      </div>
+      </form>
+     </div>
+</div>
```
Comment (su66uu) [inline template/index.html]: Here the alignment is missing? Can you make it correct?

**Review by su66uu — APPROVED** [review]

File: template/index.html (hunk 5 above)
Comment (venkykatta) [reply template/index.html]: yeah, sure

Comment (venkykatta) [reply template/index.html]: i can make it correct

**Review by su66uu — APPROVED** [review]

**Pull request merged**